server/submissionWorkflow/api.ts

```diff
@@ -0,0 +1,57 @@
+import app, { wrap } from 'server/server';
+import { ForbiddenError } from 'server/utils/errors';
+
+import { canManageSubmissionWorkflow } from './permissions';
+import {
+	createSubmissionWorkflow,
+	updateSubmissionWorkflow,
+	destroySubmissionWorkFlow,
+} from './queries';
+
+const getRequestIds = (req) => {
+	const user = req.user || {};
+	return {
+		userId: user.id,
+		communityId: req.body.communityId,
+		collectionId: req.body.id || null,
```

Comment:
You're passing in `req.body.collectionId` from your tests — I think that's what you want to be using here.